Source/WebKit/WebProcess/Plugins/PDF/PDFPluginTextAnnotation.mm

```diff
@@ -83,20 +83,20 @@ static const String cssAlignmentValueForNSTextAlignment(NSTextAlignment alignmen
     RetainPtr textAnnotation = annotation();
     bool isMultiline = [textAnnotation isMultiline];
 
-    auto element = document.createElement(isMultiline ? textareaTag : inputTag, false);
+    Ref element = document.createElement(isMultiline ? textareaTag : inputTag, false);
     element->addEventListener(eventNames().keydownEvent, *eventListener(), false);
 
-    auto& styledElement = downcast<StyledElement>(element.get());
-
     if (!textAnnotation)
         return element;
 
+    Ref styledElement = downcast<StyledElement>(element);
+
     // FIXME: Match font weight and style as well?
-    styledElement.setInlineStyleProperty(CSSPropertyColor, serializationForHTML(colorFromCocoaColor([textAnnotation fontColor])));
-    styledElement.setInlineStyleProperty(CSSPropertyFontFamily, [[textAnnotation font] familyName]);
-    styledElement.setInlineStyleProperty(CSSPropertyTextAlign, cssAlignmentValueForNSTextAlignment([textAnnotation alignment]));
+    styledElement->setInlineStyleProperty(CSSPropertyColor, serializationForHTML(colorFromCocoaColor([textAnnotation fontColor])));
+    styledElement->setInlineStyleProperty(CSSPropertyFontFamily, [[textAnnotation font] familyName]);
+    styledElement->setInlineStyleProperty(CSSPropertyTextAlign, cssAlignmentValueForNSTextAlignment([textAnnotation alignment]));
 
-    downcast<HTMLTextFormControlElement>(styledElement).setValue([textAnnotation widgetStringValue]);
+    downcast<HTMLTextFormControlElement>(styledElement)->setValue([textAnnotation widgetStringValue]);
```

Comment:
Why not just downcast this once to the right type? Could even downcast from the call to `document->createElement`